server/src/messages/dto/create-message.dto.ts

```diff
@@ -3,7 +3,12 @@ import { User } from 'src/users/entities/user.entity';
 import { Message } from '../entities/message.entity';
 
 export class CreateMessageDto extends PartialType(Message) {
+  // TODO: use real user and pass this through to the client
+  // making sure the MessageInboxView component can still fetch messages properly
+
```

Comment:
does this comment still apply? i think we are already doing this with the below `user: User` property